charts/consent/templates/deployment.yaml

```
@@ -0,0 +1,177 @@
+          {{- end }}"
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+            - name: metrics
+              containerPort: 9090
+              protocol: TCP
+          securityContext:
+            runAsUser: 2  # non-root user
+            allowPrivilegeEscalation: false
+          env:
+            - name: SENTRY_DSN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: dsn
+            - name: PROMETHEUS_ARGS
+              value: "-javaagent:/etc/prometheusjmx/prometheusjmx.jar=9090:/etc/consent-cm/prometheusJmx.yaml"
+            - name: BASIC_AUTH_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: basicAuthUser
+            - name: BASIC_AUTH_USER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: basicAuthPassword
+            - name: DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: databaseUser
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: databasePassword
+            - name: SENDGRID_API
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: sendgridApiKey
+          command: ["java"]
+          args:
+          - -Dsentry.stacktrace.app.packages=org.broadinstitute
+          - -Dsentry.dsn=$(SENTRY_DSN)
+          - -Ddw.basicAuthentication.users[0].user=$(BASIC_AUTH_USER)
+          - -Ddw.basicAuthentication.users[0].password=$(BASIC_AUTH_USER_PASSWORD)
+          - -Ddw.database.user=$(DB_USER)
+          - -Ddw.database.password=$(DB_PASSWORD)
+          - -Ddw.mailConfiguration.sendGridApiKey=$(SENDGRID_API)
+          - $(PROMETHEUS_ARGS)
+          - -jar
+          - /opt/consent.jar
+          - server
+          - /etc/consent-cm/consent.yaml
+          volumeMounts:
+            - name: configs
+              mountPath: /etc/consent-cm
+              readOnly: true
+            - name: app-secrets
+              mountPath: /etc/service-account.json
+              subPath: service-account.json
+              readOnly: true
+            - name: consent-prometheusjmx-jar
+              mountPath: /etc/prometheusjmx/prometheusjmx.jar
+              subPath: prometheusjmx.jar
+          readinessProbe:
+            httpGet:
+              path: /status
+              port: 8080
+            initialDelaySeconds: 60
+            periodSeconds: 10
+            timeoutSeconds: 1
+            failureThreshold: 10
+            successThreshold: 1
+          securityContext:
```

```suggestion
          livenessProbe:
            httpGet:
              path: /status
              port: 8080
            timeoutSeconds: 5
            initialDelaySeconds: 60
            periodSeconds: 10
            failureThreshold: 30 # Restart container after 5 minutes
          securityContext:
```
Can we add the liveness probe that will restart the pod after 5 minutes of failed status checks also?